src/org/loklak/data/ElasticsearchClient.java

```diff
@@ -949,206 +949,121 @@ public Query(final String indexName, final QueryBuilder queryBuilder, String ord
         return list;
     }
 
-    public HashMap<String, HashMap<String, Double>> classifierScore(String index, String classifierName, ArrayList<String> classes) {
-        SearchRequestBuilder request = elasticsearchClient.prepareSearch(index)
-            .setSearchType(SearchType.QUERY_THEN_FETCH)
-            .setQuery(QueryBuilders.matchAllQuery())
-            .setFrom(0)
-            .setSize(0)
-            .addAggregation(getClassifierAggregation(classifierName));
-
-        SearchResponse response = request.execute().actionGet();
+    /*
+    * All methods from here are for performing aggregation for classifiers
```

Comment:
spacing issue

IMO this shall not be mentioned. a class generally shall not be divided into sections